command/agent/config/config.go

```diff
@@ -168,6 +173,13 @@ type TemplateConfig struct {
 	StaticSecretRenderInt    time.Duration `hcl:"-"`
 }
 
+type ExecConfig struct {
+	Command            string    `hcl:"command,attr" mapstructure:"command"`
+	Args               []string  `hcl:"args,optional" mapstructure:"args"`
```

Comment:
I'm also OK with adding Directory if it is easy to support with the existing exec wrapper library. Otherwise, I'd say this is a post-beta feature.

Reply:
let me follow up in a subsequent PR